Return switching to new release note (#390)
subspace/subspace-docs: docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx

There are some cases where version updates will cause issue with your Node & Farmer and you may have to wipe & purge your node, typically when errors occur. If you have any issues you can always check our [Forums](https://forums.subspace.network) and hop in our [Discord](https://discord.gg/subspace-network) Server to ask for help.

### Wipe & Purge

> If you were running a node previously, and want to switch to a new snapshot, please perform these steps and then follow the guideline again:

Reviewer: ```suggestion
If you were running a node previously, and want to switch to a new network, please perform these steps and then follow the guide again:
```

```bash
# Replace `FARMER_FILE_NAME` with the name of the farmer file you downloaded from releases
./FARMER_FILE_NAME wipe PATH_TO_FARM
# Replace `NODE_FILE_NAME` with the name of the node file you downloaded from releases
./NODE_FILE_NAME purge-chain --chain gemini-3g
```

> Does not matter if the node/farmer executable is the previous one or from the new snapshot, both will work :)

Reviewer: ```suggestion
It does not matter if the node/farmer executable is the previous one or from the new snapshot, both will work :)
```

> The reason we require this is, with every snapshot change, the network might get partitioned, and you may be on a different genesis than the current one.

Reviewer: I'm not sure the snapshot language is as relevant as it used to be. I believe that now we are at a level of stability where a wipe/purge will only be required in extreme situations. Certainly not as often as earlier testnet stages.

Keen to hear if @nazar-pc has any thoughts on talking about snapshots like this still.

Reviewer 2: I agree, during whole 3g there were there were just a few one-offs where specific farmers had to wipe things. For everyone else there should be no issues that require starting from scratch and only transitioning to the next major version of the network requires this.

Reviewer: Let's reserve wipe/purge for switching networks or recovering from catastrophic data corruption. The former should be non-existent after mainnet but possibly useful for parallel testnets. The latter will hopefully be rare but there could be users with stability issues it will help.

In plain English, these commands are like a `reset` button for snapshot changes.

Now follow installation guide.

### Docker Wipe & Purge

In case of Docker setup run `docker compose down -v` (and manually delete custom directories if you have specified them).

> Now follow installation guide.

Reviewer: ```suggestion
Now follow the installation guide.
```

> If you are having some issues with running the node or the farmer for the subspace network, feel free to join our Discord or even better you can take a look at our Forums and review and existing questions or post a new one if needed!

Reviewer: ```suggestion
If you are having issues with running a node or farmer for Subspace, feel free to join our Discord or even better you can take a look at our forums and review existing questions or post a new one if needed!
```
